tests/em/nsem/inversion/test_complex_resistivity.py

```diff
@@ -32,7 +32,7 @@ def setUp(self):
 
         # create background conductivity model
         sigma_back = 1e-2
-        sigma_background = np.zeros(mesh.nC) * sigma_back
+        sigma_background = np.ones(mesh.nC) * sigma_back
```

Comment:
I just wanted to put an extra emphasis on this change here...

The original tests were using a 0 conductivity background model... I'm surprised they passed before this. The solver updates identified this flaw. Since fixing this to actually use the intended background conductivity, *most* of the tests still pass, except for the `phase_xx` derivatives and `phase_yy` derivatives. I'm not sure why those fail, yet all other tests pass (including derivatives for other phase components, all the amplitude components, and all of the real/imaginary components) so I suspect it is due to the model producing phases that are near the wraparound of the complex angle argument. I don't believe it's warranted to try to fix that in this PR, and it might even be fixed already by #1454, so I've marked them as expected failures for now and I'll open an issue regarding these tests.

Reply:
Nice catch @jcapriot! 

I suspect that we might not have a strong enough 3D structure to be generating large on-diagonal components, but we can handle that elsewhere